lib/printer.ts

```diff
@@ -602,7 +602,15 @@ function genericPrintNoParens(path: any, options: any, print: any) {
         parts.push(" from ");
       }
 
-      parts.push(path.call(print, "source"), ";");
+      parts.push(path.call(print, "source"));
+
+      if (n.assertions && n.assertions.length > 0) {
+          for (const assert of n.assertions) {
+              parts.push(' assert', ' {', assert.key.name, ': ', `'${assert.value.value}'`, '}');
```

Comment:
This would generate e.g. ` assert { type: 'json' } assert { foo: 'bar' }` given multiple assertions. Reading through the [parser](https://github.com/babel/babel/blob/5bbad8936b05a2b6739dcade6464dbc361118aa9/packages/babel-parser/src/parser/statement.js#L2192-L2247), it should be ` assert { type: 'json', foo: 'bar' }`. Also, I don't think we should be handling printing `Identifier` and `StringLiteral` by unwrapping them here, right? We should let recursive `print` calls handle them.

Reply:
Switched to `print` in 3d2bccb.